index.html

```diff
@@ -454,6 +454,21 @@ <h4 class="mb-0"><strong>First Contribution Hacktoberfest 2022 List</strong></h4
             </div>
           </div>
         </div>
+      </div>
+       <div class="profile-bio pt-2">
+            <div class="profile-name">Anas Qureshi</div>
+            <div class="profile-skills">Computer Science Student| Web Developer</div>
+            <hr>
+            <div class="profile-github pt-2">
+              <p>
+                <a href="https://github.com/Qureshi9764" target="_blank"><i
+                    class="fa fa-github-square fa-2x icon-link icon-link-github" aria-hidden="true"></i></a>
+                <a href="https://www.linkedin.com/in/Qureshi9764" target="_blank"><i
+                    class="fa fa-linkedin-square fa-2x icon-link icon-link-linkedin" aria-hidden="true"></i></a>
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
@Qureshi9764 
Also make sure to see if any `div` tag is missed? And the changes are working alright on localhost.